src/block/Door.php

```diff
@@ -176,4 +168,34 @@ public function getAffectedBlocks() : array{
 	private function canBeSupportedAt(Block $block) : bool{
 		return $block->getAdjacentSupportType(Facing::DOWN)->hasEdgeSupport();
 	}
+
+	public function onProjectileInteraction(Projectile $projectile) : void{
+		if($projectile instanceof WindCharge) {
+			if($this->getTypeId() === BlockTypeIds::IRON_DOOR) {
+				return;
+			}
+
+			// This is to avoid calling Door::onProjectileInteraction() twice due to two tiles.
+			if($this->top) {
```

Comment:
So, if the projectile hit the upper one, no toggle will be happened ?

Reply:
Yeah, this is my best attempt at handling this issue, if it hits both blocks the door hits, (open, close)

Reply:
Can be useful before toggling to check the other state. If its not the same as ours when projectile hits, we can ignore ? Just a theory

Reply:
feels a little hacky to be honest

Reply:
Maybe hacky but can avoid to have unexpected side effect if only the top block is hitten by the effect area.

Reply:
Yep, only checking the one half will cause bugs. It's the same as if a player were clicking the top or bottom half, so the logic should be basically the same.